lib/srv/alpnproxy/kube.go

```diff
@@ -97,6 +98,7 @@ type KubeMiddlewareConfig struct {
 	Headless     bool
 	Clock        clockwork.Clock
 	Logger       logrus.FieldLogger
+	Context      context.Context
```

Comment:
Can we give it a more specific name? I think we usually go with `CloseContext`

Reply:
Renamed https://github.com/gravitational/teleport/pull/43374/commits/9241628afbf2cb661b48a5bd8a7e1221fc447fca